zurichat/zc_main
# (Team Aristotle) Feat/centralize package management

# (Team Aristotle) Feat/centralize package management

## WHAT DOES THIS PR DO?

* centralizes package management in the project(perform dependency management for both FE and BE from the same folder)
* Fixes the error that made the backend not able to be served
* Enables developers to serve the frontend or backend from the same folder(project root) 

## HOW DOES IT DO THESE

* you can now run `yarn  zcmain:install` to install dependencies for the whole project(front and back ends), you can also install packages for FE or BE from the project root(see README for more details)
* installed a missing dependency that was making the express server not to serve
* you can now run `yarn backend:serve` or `yarn frontend:serve` as needed

## HOW DO I TEST THESE?

* clone my fork `git clone https://github.com/tobecci/zc_main.git`
* checkout to the `feat/centralize-package-management` by doing `git checkout feat/centralize-package-management`
* then test out the commands `yarn zcmain:install`, `yarn backend:serve`
* the project should serve

> Thanks for making it this far
